docs/markdown/Writing Plugins/rules-api/rules-api-subsystems.md

```diff
@@ -61,10 +60,10 @@ The subsystem should now show up when you run `./pants help shellcheck`.
 
 These classes correspond to the option types at [Options](doc:options).
 
-Every option type requires that you set the keyword argument `help`
-(and optionally specify the flag name, which defaults to the attribute name in kebab case).
-Most types require that you set `default`. You can optionally set `advanced=True` with every option
-for it to only show up with `help-advanced`.
+Every option type requires that you set the keyword argument `help`.
```

Comment:
Oh hm the `default` and `advanced` stuff looks deleted

Reply:
RIP